tests/imageupload/imageuploadprogress.js

```diff
@@ -263,4 +264,25 @@ describe( 'ImageUploadProgress', () => {
 			'</figure>]'
 		);
 	} );
+
+	it( 'should not create completeIcon element when browser is Microsoft Edge', () => {
+		env.isEdge = true;
```

Comment:
You should use `sino#stub`. And you should restore the stub after the test. You can use `testUtils` sandbox (it is automatically restored in `aftearEach`).

```js
testUtils.sinon.stub( env, 'isEdge' ).returns( true );
```